turbinia/config/turbinia_config.py

```diff
@@ -16,20 +16,57 @@
 from __future__ import unicode_literals
 
 
-"""Dummy Turbinia config file."""
+"""Turbinia Config Template"""
 
-# Turbinia Config
+##########
+# COMMON #
+##########
 
-# 'PSQ' is currently the only valid option
+# Which user account Turbinia runs as
+USER = 'turbinia'
+
+# Turbinia's installion directory
+TURBINIA_DIR = '/opt/turbinia'
+
+# Turbinia source code directory (local git repository)
+SRC_DIR = '%s/src' % TURBINIA_DIR
```

Comment:
I'm not sure how I feel about doing format strings in the config file since that would make it harder to parse from other languages.  I don't know if we'll actually do that, but I was kind of keeping it open for that.  Let's talk about this later today.

Reply:
As discussed off the line, the pythonic config is OK for now until there's a need for an universal version of config for future integration with other services in different languages.

We can explore the INI format, which is supported by Python and Go.
https://docs.python.org/2/library/configparser.html
https://github.com/go-ini/ini